seeding/user.seeding.js

```diff
@@ -68,7 +70,10 @@ export const seedUsers = async (limit, role, customValues = {}) => {
 
       const table = [];
       users.forEach((user) => {
-        table.push({ Email: user.email, Password: customValues.password || DEFAULT_PASSWORD });
+        table.push({
+          Email: user.email,
+          Password: customValues.password || USER_DEFAULTS.PASSWORD,
```

Comment:
Looking at this again, I think it is better to have a fallback password incase the user removes the password when updating the user defaults.

```suggestion
          Password: customValues.password || USER_DEFAULTS?.PASSWORD || DEFAULT_PASSWORD,
```